test/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 # Enable the tests
 
+# Allow the source files to find headers in src/
+include_directories(${PROJECT_SOURCE_DIR}/src)
```

Comment:
you shouldn't need this. the only extra thing here is the time unit which is already in include/benchmark.